test/performance_test.js

```diff
@@ -49,49 +48,7 @@ describe("Rendering performance optimizations", () => {
     });
   });
 
-  describe("ArrayField", () => {
-    const onChange = () => {};
-    const onBlur = () => {};
-    const schema = {type: "array", items: {type: "string"}};
-    const uiSchema = {};
-    const registry = getDefaultRegistry();
-
-    it("should not render if next props are equivalent", () => {
-      const props = {
-        registry,
-        schema,
-        uiSchema,
-        onChange,
-        onBlur
-      };
-
-      const {comp} = createComponent(ArrayField, props);
-      sandbox.stub(comp, "render").returns(<div/>);
-
-      setProps(comp, props);
-
-      sinon.assert.notCalled(comp.render);
-    });
-
-    it("should not render if next formData are equivalent", () => {
-      const props = {
-        registry,
-        schema,
-        formData: ["a", "b"],
-        onChange,
-        onBlur
-      };
-
-      const {comp} = createComponent(ArrayField, props);
-      sandbox.stub(comp, "render").returns(<div/>);
-
-      setProps(comp, {...props, formData: ["a", "b"]});
-
-      sinon.assert.notCalled(comp.render);
-    });
-  });
```

Comment:
The "shouldComponnetUpdate" of "ObjectField" and "ArrayField" were guaranteed by "SchemaField". So, I thought those two function were no longer necessary and the related tests were removed.